src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -651,6 +652,7 @@ public function thisProductHasAnImageWithACode(ProductInterface $product, $image
 
     /**
      * @Given /^(this product) belongs to ("([^"]+)" shipping category)$/
+     * @Given product :product has its shipping category changed to :shippingCategory
```

Comment:
product :product shipping category has been changed to :shippingCategory (to be consistent with step added above)?